package.json

```diff
@@ -59,13 +60,16 @@
     "@babel/plugin-proposal-optional-chaining": "^7.2.0",
     "@babel/preset-env": "^7.5.5",
     "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
     "coveralls": "^3.0.6",
     "eslint": "^6.1.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.18.2",
     "mocha": "^6.2.0",
     "mocha-lcov-reporter": "^1.3.0",
     "nodemon": "^1.19.1",
-    "nyc": "^14.1.1"
+    "nyc": "^14.1.1",
+    "sinon": "^7.4.1",
```

Comment:
Why `sinon`?

Reply:
It is used to check for what a function returns when it is called. Kind of faking a function.

Reply:
OK.